src/config.rs

```diff
@@ -74,22 +74,7 @@ impl Config<M31ExtConfig> {
             grinding_bits: 10,
             polynomial_commitment_type: PolynomialCommitmentType::Raw,
             fs_hash: FiatShamirHashType::SHA256,
-            gkr_config: M31ExtConfig,
-            gkr_scheme,
-        }
-    }
-}
-
-impl Config<BN254Config> {
-    pub fn new(gkr_scheme: GKRScheme) -> Self {
-        Config {
-            field_size: 93,
```

Comment:
oh i realized this two shouldn't be merged. and the BN254Config isn't implemented correctly.

Reply:
```
            field_size: 254,
            security_bits: 100,
            #[cfg(feature = "grinding")]
            grinding_bits: 0,
```

Reply:
yikes - I stepped on some landmine here.  I will revert my change and set proper params for `BN254Config` here